modules/local/check_input_vcf.nf

```diff
@@ -0,0 +1,32 @@
+// Import generic module functions
+include { saveFiles } from './functions'
+
+params.options = [:]
+
+process CHECK_INPUT_VCF {
+    tag "check_vcf"
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:'genome', meta:[:], publish_by_meta:[]) }
+
+    conda (params.enable_conda ? "conda-forge::python=3.8.3" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/python:3.8.3"
+    } else {
+        container "quay.io/biocontainers/python:3.8.3"
+    }
+
+    input:
+    path vcfs
+
+    output:
+    path '*.txt'       , emit: txt
+
+    script: // This script is bundled with the pipeline, in nf-core/raredisease/bin/
+    def input = vcfs.collect { it.toString() }
+    """
+    check_input_vcf.py \\
+        --INPUT_VCF ${input.join(' ')} \\
+        --OUTPUT unprocessed_vcfs.txt
```

Comment:
@ramprasadn I don't recall but did we decide against using `bcftools view` here over the python script? :)

Reply:
If we use bcftools view, then we would have to use something like grep and awk to get the output in the desired format, right?